src/addressResolvers/lens.ts

```diff
@@ -25,13 +36,29 @@ export async function lookupAddresses(addresses: Address[]): Promise<Record<Addr
               }
             }
           }
-        `
-      }
-    });
+        `);
 
     return Object.fromEntries(items.map(i => [i.ownedBy, i.handle])) || {};
   } catch (e) {
     capture(e);
     return {};
   }
 }
+
+export async function resolveName(handle: string): Promise<string | undefined> {
```

Comment:
But `params` passed in `src/api.ts` is an array 🤔

Reply:
params is a string in the case of `resolve_name`

Reply:
like this? 
```JSON
{
    "method": "resolve_name",
    "params": "chaitu.eth"
}
```

Doesn't work, i get `ResolutionError: Domain chaitu.eth is not supported`

Reply:
Anyway i believe it should be an array if it is named `params`

Reply:
Fixed, api should work now.

What var name would be appropriate ?

Reply:
I would still keep it array and limit it to length of one 😄 I checked eth RPC calls https://composer.alchemy.com/ and there are many requests that use `params` with a single item 

else maybe `input` or `name` is more relevant